tests/features/tmgmt/tmgmt_poetry.feature

```diff
@@ -17,6 +17,26 @@ Feature: TMGMT Poetry features
       | de        |
       | it        |
 
+  Scenario Outline: Checking automatic counter request.
+    Given I am logged in as a user with the 'administrator' role
+    And I go to "node/add/page"
+    And I fill in "Title" with "<title>"
+    And I fill in "Body" with "<body>"
+    And I press "Save"
+    And I select "Published" from "state"
+    And I press "Apply"
+    Then I click "Translate" in the "primary_tabs" region
+    And I select the radio button "" with the id "edit-languages-fr"
+    And I press "Request translation"
+    And I press "Submit to translator"
+    And I store the job reference of the translation request page
+    Then the poetry translation service received the translation request
```

Comment:
I would first add the step where the counter is received, as IRL you can't receive a translation if you have not first received the counter.
You should check also that the counter needs to be received only the first time with part 0, the second time, the part will increase +1.